makefile

```diff
@@ -0,0 +1,12 @@
+default:
+	cat makefile
+get_synthetic_data:
+	wget -O synthetic_data.csv https://gist.github.com/Niarfe/25100a860cbb660150a3fec87378d8ac/raw/
+square_wave: synthetic_data.csv
```

Comment:
the file will work as is, however, just for readability can we add a space to the line above the job names?  👍